scripts/test-sim-dev.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+CONTAINER_NAME=storj_sim_postgres
+docker run -d --rm -p 5433:5432 --name $CONTAINER_NAME postgres:9.6
+
+cleanup(){
+  docker rm -f $CONTAINER_NAME
+}
+trap cleanup EXIT
+
+STORJ_SIM_DATABASE=${STORJ_SIM_DATABASE:-"teststorj"}
+
+RETRIES=5
+
+until docker exec $CONTAINER_NAME psql -h localhost -U postgres -d postgres -c "select 1" > /dev/null 2>&1 || [ $RETRIES -eq 0 ]; do
+  echo "Waiting for postgres server, $((RETRIES--)) remaining attempts..."
+  sleep 1
+done
+
+docker exec $CONTAINER_NAME psql -h localhost -U postgres -c "create database $STORJ_SIM_DATABASE;"
+
+export STORJ_SIM_POSTGRES="postgres://postgres@localhost:5433/$STORJ_SIM_DATABASE?sslmode=disable"
```

Comment:
maybe we could export STORJ_POSTGRES_TEST too? i think something still uses that instead of STORJ_SIM_POSTGRES

Reply:
STORJ_POSTGRES_TEST is a variable to run unit tests with postgres. I think it's not applicable here. I'm thinking about refactoring this script to be generic way to run something with postgres but that will be next PR.